mesop/cli/cli.py

```diff
@@ -71,6 +85,12 @@ def main(argv: Sequence[str]):
   if not FLAGS.prod:
     enable_debug_mode()
 
+  if FLAGS.reload_demo_modules:
```

Comment:
[nit] May want to add a comment on why we need to do this for demo modules since I don't think it would be intuitive to people new to the code.

Reply:
added a comment.